pa11ycrawler/pipelines.py

```diff
@@ -182,6 +221,7 @@ def process_item(self, item, spider):
                 )
             )
 
+        self.check_title_match(item['page_title'], stdout, spider.logger)
```

Comment:
I saw the logging and that makes sense. Let me re-word my question though...if an error is encountered in this line, will pa11y results still be written out [here](https://github.com/edx/pa11ycrawler/pull/12/files#diff-6acf4a4e741c51fe7c49e76792ec73e9L186)? For what page?